pages/outreachy.md

```diff
@@ -54,8 +57,56 @@ Interested in becoming a mentor? Contact one of the coordinators!
 
 ## Available tasks
 
-Coming soon. At this moment, we recommend looking at the following issue trackers for smaller tasks that might
-give a sense of what's coming:
+### Instrumentation for mobile applications
+
+Mentor: Gary Brown
+
+* Collect ideas on what's possible and what's desirable for mobile applications
+* Decide and experiment with different options for an optimized mobile experience, like, which sampling strategies
+make more sense
+* Create the missing pieces, like OpenTracing framework integration for some Android component, or a specific Jaeger
+tracer for mobile usage
+* Documentation + examples
+* The end result would be: a trace starts when a user opens the app on the phone and backend microservices are called
+
+### Split the JavaScript OT library: NodeJS and Browser
+
+Mentor: Pavol Loffay
+
+* HttpSender which works in a web browser
+* Intercept web browser events (maybe can be done as OpenTracing instrumentation)
+
+### Drive consensus on a common format for reporting trace/span data
+
+Mentor: Gary Brown
+
+### OpenTracing MockTracer JUnit rule.
+
+Mentor: Pavol Loffay
+
+* Clear reported spans between tests. Automatically assert on errors.
+
+### Move community Jaeger tracers to the jaegertracing organization.
+
+Mentor: Gary Brown
+
+* This task include writing xdock tests, missing reporters..
+
+### Create OpenTracing API for language not currently supported
+
+Mentor: Pavol Loffay
+
+* For instance, for missing Android parts.
+
+### Write Jaeger tracer implementation for any OpenTracing API.
+
+Mentor: Gary Brown
+
+* C#
+* Kotlin for Android
```

Comment:
This is funny there is no OpenTracing Kotlin API

Reply:
I believe that Kotlin is a JVM-based language, so, the Java one would work. I'm not sure Kotlin would make sense here, but it's part of the task to figure this out.